source/crud/tests/write/bulkWrite-arrayFilters.yml

```diff
@@ -4,42 +4,65 @@ data:
-            name: "bulkWrite"
-            arguments:
-                requests:
-                    -
-                        # UpdateOne when one document matches arrayFilters
-                        name: "updateOne"
-                        arguments:
-                            filter: {}
-                            update:
-                                $set: {"y.$[i].b": 2}
-                            arrayFilters:
-                              - {i.b: 3}
-                    -
-                        # UpdateMany when multiple documents match arrayFilters
-                        name: "updateMany"
-                        arguments:
-                            filter: {}
-                            update:
-                                $set: {"y.$[i].b": 2}
-                            arrayFilters:
-                              - {i.b: 1}
-                options: { ordered: true }
-        outcome:
-            result:
-                deletedCount: 0
-                insertedCount: 0
-                insertedIds: {}
-                matchedCount: 3
-                modifiedCount: 3
-                upsertedCount: 0
-                upsertedIds: {}
-            collection:
-                data:
-                    - {_id: 1, y: [{b: 2}, {b: 2}]}
-                    - {_id: 2, y: [{b: 0}, {b: 2}]}
+  -
+    description: "BulkWrite with arrayFilters"
+    operations:
+      -
+        name: "bulkWrite"
+        arguments:
+          requests:
+            -
+              # UpdateOne when one document matches arrayFilters
+              name: "updateOne"
+              arguments:
+                filter: {}
+                update: { $set: { "y.$[i].b": 2 } }
+                arrayFilters: [ { "i.b": 3 } ]
+            -
+              # UpdateMany when multiple documents match arrayFilters
+              name: "updateMany"
+              arguments:
+                filter: {}
+                update: { $set: { "y.$[i].b": 2 } }
+                arrayFilters: [ { "i.b": 1 } ]
+          options: { ordered: true }
+        result:
+          deletedCount: 0
+          insertedCount: 0
+          insertedIds: {}
+          matchedCount: 3
+          modifiedCount: 3
+          upsertedCount: 0
+          upsertedIds: {}
+    expectations:
+      -
+        command_started_event:
+          command:
+            update: *collection_name
+            updates:
+              -
+                q: {}
+                u: { $set: { "y.$[i].b": 2 } }
+                multi: false
+                upsert: false
+                arrayFilters: [ { "i.b": 3 } ]
+              -
+                q: {}
+                u: { $set: { "y.$[i].b": 2 } }
+                multi: false
+                upsert: false
+                arrayFilters: [ { "i.b": 1 } ]
+            ordered: true
+            writeConcern: { $exists: false }
+            bypassDocumentValidation: { $exists: false }
+            command_name: update
```

Comment:
Can you clarify how this and the next `database_name` key are supposed to be used?